README.md

```diff
@@ -201,6 +201,7 @@ The DOF column denotes the number of actuated degrees of freedom.
 | Name                          | Robot                 | DOF | Format     |
 |-------------------------------|-----------------------|-----|------------|
 | `double_pendulum_description` | Double Pendulum       | 2   | URDF       |
+| `dynamixel_2r_mj_description` | Dynamixel 2R          | 1   | MJCF       |
```

Comment:
Guess it's 2 DOFs here :slightly_smiling_face:

Reply:
Corrected in 7448edb0fb6e46a42635c3b89705b9a52aeaec84, thanks :+1: